domains/authorization/Authorization.API/API/Authorization/Controllers/ApiModels.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace API.Authorization.Controllers;
+
+public record AuthorizationUserRequest(
+    [property: JsonPropertyName("sub")] string Sub,
+    [property: JsonPropertyName("orgId")] string OrgId,
+    [property: JsonPropertyName("name")] string Name
+);
+
+public record AuthorizationClientRequest(
+    [property: JsonPropertyName("sub")] string Sub
+);
+
+public record AuthorizationResponse(string Sub, string Name, string SubType, IEnumerable<string> OrgIds, string Scope);
```

Comment:
I'm wondering - in the case with a MitID user, we need a claim with the organization name. But do we need the organization name in the response as well, or do we just use the org name claim from MitId directly?